charts/warpstream-agent/README.md

```diff
@@ -1,16 +1,64 @@
+- **Object Storage Access**: How will your pods authenticate requests to your object storage?
+- **Authentication**: How will your clients securely connect with your WarpStream cluster?
+
+### Networking
+As with any Kubernetes deployment, there are a variety of network topologies available, but two will be highlighted here.
+1. **Direct Route** - a network route exists for clients to connect directly with each pod
+
+In this scenario, the deployment utilizes the built-in load-balancing provided by the WarpStream Service Discovery to round-robin client connections to the available pods. Learn more about WarpStream's custom service discovery in the [Service Discovery](https://docs.warpstream.com/warpstream/overview/architecture/service-discovery#kafka-service-discovery) documentation. In this configuration, each pod will need to be reachable from the clients via the internal IP which it advertises as part of the Service Discovery process.
+
+2. **Load Balancer/Proxy** - clients connect to the agents via a traditional network load balancer or proxy
+
+In this scenario, because the WarpStream agents are entirely stateless, they can be deployed behind a network load balancer. This means clients only need to be able to reach the load balancer. The key to this configuration is that all agents need to advertise the hostname or IP of the load balancer instead of their local IP. Read more about this option [here](https://docs.warpstream.com/warpstream/byoc/advanced-agent-deployment-options/configure-warpstream-agent-within-a-container-or-behind-a-proxy).
+
+
+### Scaling
+WarpStream clusters can auto-scale easily due to the stateless nature of agents. The recommendation is to auto-scale based on CPU usage with a 50% average usage target. Auto-scaling is disabled by default in the Helm chart (3 replicas are used instead). See `autoscaling` in [values.yaml](values.yaml) for the available options.
 
-### Install the latest stable release
+### Object Storage Access
+WarpStream agents rely on object storage as opposed to local disks. This means each pod needs permissions to access your object storage bucket. Configuration varies depending on your cloud provider. See [Object Storage Configuration](https://docs.warpstream.com/warpstream/byoc/deploy/different-object-stores) for more details.
+
+Other useful references:
+- https://docs.aws.amazon.com/eks/latest/userguide/service-accounts.html
+- https://azure.github.io/azure-workload-identity/docs/introduction.html
+- https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity
+
+### Authentication
+WarpStream supports secure authentication and encryption via SASL and TLS. Authentication is not enabled by default in the Helm chart. Enabling it is highly recommended, especially if agents are exposed outside your VPC. See the [Authentication](https://docs.warpstream.com/warpstream/byoc/advanced-agent-deployment-options/enable-agent-auth) documentation for more details.
+
+
+## Quickstart
```

Comment:
Can you move quickstart higher? right after Prerequisites. So the flow is:

1. Install
2. Prereqs
3. Quick Start
4. More detailed information